apps/health_campaign_field_worker_app/lib/router/app_router.dart

```diff
@@ -190,7 +188,6 @@ class AppRouter extends _$AppRouter {
                     page: HouseholdAcknowledgementRoute.page,
                     path: 'household-acknowledgement',
                   ),
-                  AutoRoute(page: ChecklistViewRoute.page, path: 'view'),
```

Comment:
Why is this removed? Survey form is been used in registration delivery as well right?

Reply:
In registration delivery that router path is not there that is why we removed from here as well.